sumologic_collectd_metrics/metrics_config.py

```diff
@@ -35,7 +35,8 @@ class ConfigOptions(object):
     # Static option, not configurable yet. Default is application/vnd.sumologic.carbon2
     content_type = 'ContentType'
     shutdown_max_wait = "ShutdownMaxWait"  # seconds
-
+    # Usage Tracking header
+    x_sumo_client = 'XSumoClient'
```

Comment:
I don't think this needs to (or should be) user-configurable

Reply:
agreed